.gitignore

```diff
@@ -6,3 +6,7 @@ out*
 MYMETA*
 Makefile
 install_tmp
+perl/docs.tar.gz
```

Comment:
Perl docs should be committed, before a release it should be rebuilt via `prerelease.sh`